src/System.Management.Automation/utils/PSTelemetryMethods.cs

```diff
@@ -196,12 +196,12 @@ internal static void ReportModuleLoad(PSModuleInfo foundModule)
             System.Management.Automation.Host.TranscriptionData transcriptionData)
         {
             bool isConstrained = (iss != null) && (iss.DefaultCommandVisibility != SessionStateEntryVisibility.Public) && (iss.LanguageMode != PSLanguageMode.FullLanguage);
-            bool isTranscripting = (transcriptionData != null) && (transcriptionData.SystemTranscript != null);
+            bool isTranscribing = (transcriptionData != null) && (transcriptionData.SystemTranscript != null);
 
             TelemetryWrapper.TraceMessage("PSNewLocalSession", new
             {
                 Constrained = isConstrained,
-                Transcripting = isTranscripting
+                Transcribing = isTranscribing
```

Comment:
The change would be correct English, but it doesn't matter, this data is being ingested, so changing the property name would mess up the data - we should reject this change.